site/src/component/RecentOfferings/RecentOfferings.scss

```diff
@@ -12,7 +12,7 @@
   width: 100%;
   max-width: 540px;
   .bi-check {
-    transform: scale(1.5) translateY(-1px);
+    transform: scale(1.5);
   }
```

Comment:
I think we should remove this entirely, as `bi-check` is a class name for **b**ootstrap **i**cons